corehq/apps/hqwebapp/templates/password_change.html

```diff
@@ -1,12 +1,12 @@
 {% extends base_template %}
 {% block title %}Change User Password{% endblock %}
 {% block content %}
-<h2>Change Password for {{user.username}}</h2>
-<form method="POST">
-	{% csrf_token %}
+  <h2>Change Password for {{user.username}}</h2>
```

Comment:
I know this isn't part of this PR, but this should be translated. (This change makes everything so much easier to see :smile:  )

Reply:
✅